coop_env.py

```diff
@@ -88,7 +89,7 @@ def _get_total_reward(self):
     def _step(self, actions):
         """Execute the specified list of actions.
         """
-        assert self.action_space.contains(actions)
+        # assert self.action_space.contains(actions)
```

Comment:
I debugged this. The types are wrong. The function expects a tuple containing 2 arrays, but is getting an array of 2 arrays.